types/shared.d.ts

```diff
@@ -2,11 +2,17 @@ export declare type LoadConnect = () => Promise<StripeConnectWrapper>;
 
 export declare type OverlayOption = "dialog" | "drawer";
 
+/**
+ * UI configuration options for the Connect instance.
+ */
 export declare type UIConfigOptions = {
   overlay?: OverlayOption;
   overlayZIndex?: number;
 };
 
+/**
+ * Appearance options for the Connect instance.
+ */
 export declare type AppearanceOptions = {
   colorPrimary?: string;
```

Comment:
I will file a task to add JSDOCs for these - we should do it in our codebase and then port them over